src/main/java/com/github/maxopoly/WurstCivTools/ConfigParser.java

```diff
@@ -57,6 +54,10 @@ public void parseTools(ConfigurationSection config) {
 			WurstEffect effect;
 			switch (type) {
 			case "PYLONFINDER":
+				if (current.getBoolean("enabled",false) == false){
```

Comment:
Having disabled by default was meant to address malformed configs. On my dev env I don't have FM installed, so having an easy way to disable stuff was nice.